src/utils/check-pr/check-pr.py

```diff
@@ -274,6 +320,9 @@ def run(self, *, merge_on_success: bool = False) -> None:
         )
 
         self.deploy(release_filename)
+
+        self.register()
```

Comment:
add a parameter to check if we want this behavior 
i suggesst `--unattended`

Reply:
Done.